calico/felix/selectors.py

```diff
@@ -0,0 +1,602 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Metaswitch Networks
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+selectors
+~~~~~~~~~
+
+Parser for selector expressions, which can match against dicts of labels.
+
+For example, the dict {"a": "b"}, would match a selector such as:
+
+    a in {"b", "c"}
+
+or
+
+    a == "b"
+
```

Comment:
Where do I look to see an complete account of the supported selector syntax?  (Right here would feel like a good place.)